distribution/docs/src/main/resources/_contents/_catalog-contents/extending-catalog-contents.adoc

```diff
@@ -1255,7 +1255,8 @@ This plugin can be installed and uninstalled using the normal processes describe
 No configuration is necessary for this plugin. 
```

Comment:
It seems okay when reading it all in context. It might be more accurate to document no configuration is required. But I don't think that adds a ton of value from what is there.